app/models/user.rb

```diff
@@ -181,6 +181,11 @@ def assistantship(course)
     Assistantship.find_by(user_id: self, course_id: course)
   end
 
+  # Checks if the user is only a student. Returns false if the user is a student and also an assistant in any course
```

Comment:
Maybe: Returns whether user is only student system wide (not assistant or admin)